src/OrbitGl/ThreadTrack.cpp

```diff
@@ -69,22 +68,28 @@ std::string ThreadTrack::GetName() const {
   ;
 }
 
-void ThreadTrack::InitializeNameAndLabel() {
+constexpr std::string_view kAllThreads = " (all_threads)";
+std::string ThreadTrack::GetLabel() const {
+  auto thread_id = GetThreadId();
+  auto name = GetName();
+  if (thread_id == orbit_base::kAllThreadsOfAllProcessesTid) {
+    return name;
+  } else if (thread_id == orbit_base::kAllProcessThreadsTid) {
+    return name.append(kAllThreads);
```

Comment:
The label here was `process_name.append("(all_threads)")` before, but now it becomes `"All Threads (all_threads)"`.

Reply:
Are we showing the full label in the tooltip of the thread track tab?

Reply:
> The label here was process_name.append("(all_threads)") before, but now it becomes "All Threads (all_threads)".

Nice catch!

Reply:
> Are we showing the full label in the tooltip of the thread track tab?

I don't know what you mean, I'm not changing tooltip at all. But I've just found that all thread track tab is showing a horrible tooltip. I will make a bug there

Reply:
I mean currently if the thread track label is too long, we only show part of it. Should we consider showing the complete content in the tab tooltip? Therefore if the user point to a thread track tab, he will able to see it.

Reply:
Could be... But all track labels are being shortened if they are too long. The only difference with the Thread tracks ones if that we prioritise some trailing characters in the label we draw.

You might be right but in this case I will propose to add all track labels in the tooltips. What do you think?

Reply:
Yes, it makes sense. But it would affect all tracks, not only Thread Track. Maybe it also might be fixed if we limit the size of the text.